docs/fundamentals/apicompat/preview-apis.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Preview APIs
+description: Provides an understand how APIs can be marked as preview.
+ms.date: 4/4/2024
+---
+
+# Preview APIs
+
+The .NET platform prides itself with taking compatibility very seriously. As
+such, the the library ecosystem tends to avoid making breaking changes,
+especially with respect to the API.
+
+Nonetheless, when designing APIs it's important to collect feedback from users
+and change the API based on their feedback. To avoid surprises, it's important
+that users understand which APIs are considered stable and which ones are still
+in active development and might change.
+
+There are multiple ways an API can express to be in preview form:
+
+* The entire component is considered preview:
+    - Exposed in a preview release of the .NET runtime
+    - Exposed in a prerelease NuGet package
+* An otherwise stable component specifically marks certain APIs as preview:
+    - Marked with <xref:System.Runtime.Versioning.RequiresPreviewFeaturesAttribute>
+    - Marked with <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>
```

Comment:
```suggestion
- The entire component is considered preview:

    - Exposed in a preview release of the .NET runtime
    - Exposed in a prerelease NuGet package

- An otherwise stable component specifically marks certain APIs as preview:

    - Marked with <xref:System.Runtime.Versioning.RequiresPreviewFeaturesAttribute>
    - Marked with <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>
```